README.adoc

```diff
@@ -22,6 +22,8 @@ to modern, idiomatic, functional-style Java and to use Elliptic Curve types from
 
 The API is distributed as an API-only JAR (```secp-api-_version_.jar```) and there will be multiple implementations of the API. The API JAR currently requires JDK 9 or later but we may backport it to JDK 8 in a future release depending upon feedback from the community.
 
+For details see the https://msgilligan.github.io/javadoc/secp256k1-jdk/0.2[Javadoc].
```

Comment:
Shall we add Javadocs to `bitcoinj.org` (probably separate PR)?

Reply:
Yes, see Issue #243